### PR TITLE
Shifting code from staging-site

### PR DESCRIPTION
**Pull request by JacOng17**

### CODE REVIEW

File: _insights/insights.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+The system-level conditions (resources, ownership, authority) for innovation work are persistently precarious and often short term for projects and labs/innovation teams. While this can be a helpful driver for innovation, in a public sector context innovation work is often grappling with complex long-term issues.
+
+But short term funding cycles often leads to time and effort being diverted towards justifying continued support and funding and away from the work itself. It can create pressures to focus on quick wins and justifying the work, over system-level change and a longer-term focus.
+
+#### How might we…
+
+- Develop approaches to funding and resourcing labs and innovation teams that incentivise long term thinking and outcomes?
+- Better demonstrate value and impact in the short term that supports a long term approach?
+
+(NOTE - here's a couple more to added here too)
```
Comment (mermop) [inline _insights/insights.md:82]: Placeholder alert!

Comment (JacOng17) [reply _insights/insights.md:82]: :stuck_out_tongue: Haha, good to note! :+1:  Thanks @mermop

File: _insights/lab-journey.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+---
+title: Service Innovation Lab journey
+subtitle: How the lab adapted and evolved its approach over three years
+permalink: /insights/lab-journey/
+excerpt: How the lab adapted and evolved its approach over three years
+image: /assets/img/insights/insights-jounrey-bg.png
+image_accessibility:  icon representing a journey
+external_url: "[Project repo](url)"
```
Comment (mermop) [inline _insights/lab-journey.md:8]: Placeholder link 👀

File: _insights/labtips.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
+---
+title: Practical tips on running a lab
+subtitle: Reflections from the lab team about key areas to think about when developing and running a lab
+permalink: /insights/lab-tips/
+excerpt: Reflections from the lab team about key areas to think about when developing and running a lab
+image: /assets/img/insight/insight-bg.png
+image_accessibility: incon for offering advice
+external_url: "[Project repo](url)"
```
Comment (mermop) [inline _insights/labtips.md:8]: Placeholder link 👀

File: _layouts/about-page.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            {% endfor %}
+          </ul>
+        {% endif %}
+      </ul>
+    </aside>
+    {% endif %}
+
+    <div class="nz-grid nz-width-two-thirds">
+      <section class="nz-section page-tag-results-content">
+        <!-- <div class="small-caps small-caps-no-margin">Background</div>
```
Comment (mermop) [inline _layouts/about-page.html:115]: Can probably delete these commented out lines

File: _insights/labtips.md (hunk 3 above)
Comment (mermop) [reply _insights/labtips.md:8]: There are quite a few of these actually - do they have a function other than as a placeholder?

File: pages/projects.md [hunk 5]
```diff
@@ -17,27 +17,39 @@ h2 {
...
 <div class="nz-grid">
+  <button class="categories-button-clear filter-button active-filter">All</button>
+  <button class="categories-button filter-button" data-category="integrated-services">Integrated Services</button>
+  <button class="categories-button filter-button" data-category="tech">Tech</button>
+  <button class="categories-button filter-button" data-category="ways-of-working">Ways of working</button>
+  <button class="categories-button filter-button" data-category="partnerships">Partnerships</button>
+</div>
+
+<div class="child-page-listing nz-grid-full">
+<!-- <div class="nz-grid"> -->
```
Comment (mermop) [inline pages/projects.md:34]: As above - these commented out lines can be deleted

File: pages/process.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+content_wide: true
+content_focus: false
+gridless: true
+---
+
+<section class="nz-section background-gray">
+  {% include tools-process-nav.html %}
+
+  <div class="nz-grid">
+    <h2 tabindex="0" style="color: #2337CE">Process</h2>
```
Comment (mermop) [inline pages/process.md:15]: The inline style isn't ideal here but we probably don't need to optimise for maintainability here 😄

File: pages/home.html [hunk 7]
```diff
@@ -8,8 +8,6 @@
   <div class="hero" role="main">
     <div class="vertical" role="main">
       <h1 class="heading">Welcome to<br />The Service Innovation Lab</h1>
-      <a class="btn btn-primary btn-lg" role="button" href="{{ site.baseurl }}/tools">Toolkit</a>
     </div>
-    </body>
```
Comment (mermop) [inline pages/home.html:13]: Weird that this was here!

Comment (JacOng17) [reply pages/home.html:13]: Oh yup!

**Pull request merged**